pkg/schema/openapi.go

```diff
@@ -52,17 +52,29 @@ func (o *OpenAPIDocument) calculateProperties(schemaVal interface{}) *yamlmeta.M
 		property := yamlmeta.Map{Items: []*yamlmeta.MapItem{
 			{Key: "type", Value: "object"},
 			{Key: "additionalProperties", Value: false},
-			{Key: "properties", Value: &yamlmeta.Map{Items: properties}},
 		}}
+
+		if typedValue.GetDescription() != "" {
+			property.Items = append(property.Items, &yamlmeta.MapItem{Key: "description", Value: typedValue.GetDescription()})
```

Comment:
Given that we're using the same string in multiple places, should we extract the OpenAPI property names into constants?